packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx

```diff
@@ -0,0 +1,123 @@
+import { cssObj } from '@fuel-ui/css';
```

Comment:
few changes to make this component closer to figma

- title should be black (intentsBase 0 or 12)
- project icon icon is looking narrow. width and height should be the same
- url black, and smaller
- can we customizem a bit <Tag> style to reach more similar style?
- whole card should be clickable(and pointer cursor) instead of only url.

![image](https://github.com/FuelLabs/fuels-portal/assets/8636507/5e982c4b-5c10-4afb-ab49-ef9d3f687166)